packages/chakra-ui/src/CheckboxesWidget/CheckboxesWidget.tsx

```diff
@@ -55,13 +53,13 @@ export default function CheckboxesWidget<
       invalid={rawErrors && rawErrors.length > 0}
       {...(chakraProps as any)}
     >
+      <FieldsetLegend>{label}</FieldsetLegend>
```

Comment:
The `labelValue(label, hideLabel || !label)` you are removing below is designed to hide the label when it is not present or being explicitly hidden. You need to account for that here. Can you make sure the tests cover all three use-cases?
```suggestion
      if (!hideLabel && label) {
        <FieldsetLegend>{label}</FieldsetLegend>
      }
```